sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/Checkpoint.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.EventHubs.Core;
+using System;
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    /// <summary>
+    ///   Contains all the information needed to store the state of an <see cref="IPartitionProcessor" />.
+    /// </summary>
+    ///
+    public class Checkpoint
+    {
+        /// <summary>
+        ///   The path of the specific Event Hub this checkpoint is associated with, relative
```

Comment:
nit:  There's some inconsistency between the comment, the name, and members of other client types in the library.   I believe `EventHubName` is more accurate for the semantics of this, so I opened the following to unify things:

https://github.com/Azure/azure-sdk-for-net/issues/7096

Reply:
Changed from "path" to "name" in the Processor namespace comments to avoid more work in the future.